OpenRA.Mods.Common/Projectiles/GravityBomb.cs

```diff
@@ -36,6 +36,8 @@ public class GravityBombInfo : IProjectileInfo
 
 		public readonly WDist Speed = WDist.Zero;
 
+		public readonly WDist ForwardSpeed = WDist.Zero;
+
 		[Desc("Value added to speed every tick.")]
 		public readonly WDist Acceleration = new WDist(15);
```

Comment:
You can then also make this a `WVec` to address @GraionDilach's request (which seems reasonable to me).